modules/oe_webtools_laco_widget/oe_webtools_laco_widget.module

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * @file
+ * OpenEuropa Webtools Laco Widget module.
+ */
+
+declare(strict_types = 1);
+
+/**
+ * Implements hook_page_attachments().
+ */
+function oe_webtools_laco_widget_page_attachments(array &$attachments) {
+  $logger = \Drupal::logger('oe_webtools_laco_widget');
+  $config = \Drupal::config('oe_webtools_laco_widget.settings');
+
+  $include = $config->get('include');
+  if (!$include) {
+    $logger->info('There are no selectors to include the Laco widget for. Widget is not being printed.');
```

Comment:
The purpose of this message is to alert the site developer that the module should not be enabled at all if no include is specified, shall we add this as a suggestion to the message? We could also promote this to a warning.